benches/benches/vm_initialization.rs

```diff
@@ -99,8 +104,36 @@ pub fn vm_initialization(c: &mut Criterion) {
                     Interpreter::<_, Script, NotSupportedEcal>::with_memory_storage(),
                 );
                 let ready_tx = tx.clone().test_into_ready();
-                black_box(vm.init_script(ready_tx))
-                    .expect("Should be able to execute transaction");
+
+                // Initialize the VM and require the allocation of the whole memory
+                // to charge for the worst possible case.
+                black_box({
```

Comment:
I could say a the same for a lot of this. Can we move everthing in these curlies into some helper?